components/reddit/sources/new-hot-post-on-a-subreddit/new-hot-post-on-a-subreddit.js

```diff
@@ -0,0 +1,184 @@
+const reddit = require("../../reddit.app.js");
+
+module.exports = {
+  key: "new-hot-spot-on-a-subreddit",
+  name: "New hot spot on a subreddit",
+  description:
+    "Emits an event each time a new host post is added to the top 10 a subreddit.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    reddit,
+    subreddit: {
+      type: "string",
+      label: "Subreddit",
+      description: "The subreddit you'd like to watch for new hot posts.",
+      default: "redditdev",
+    },
+    g: {
+      type: "string",
+      label: "g",
```

Comment:
I'd make the label read "Locale" and change the description to:

"Hot posts differ by region, and this refers to the locale you'd like to watch for hot posts. Refers to the **g** param in the Reddit API".